hack/verify-gofmt.sh

```diff
@@ -24,9 +24,9 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 GO_VERSION=($(go version))
 
-if [[ -n $(echo "${GO_VERSION[2]}" | grep -E 'go1.1|go1.2|go1.3') ]]; then
+if [[ -n $(echo "${GO_VERSION[2]}" | grep -E 'go1.1|go1.2|go1.3|go1.4|go1.6') ]]; then
```

Comment:
also agreed that the logic was wrong. this was making the check simply not run if the go version was <1.4 (since gofmt rules changed). now it fails basically every go version.